tensilelite/Tensile/TensileInstructions/Instructions.py

```diff
@@ -2359,6 +2365,11 @@ def __init__(self, dst, src0, src1, vop3: Optional[VOP3PModifiers] = None, comme
         super().__init__(CvtType.CVT_PK_F32_to_FP8, dst, [src0, src1], vop3, comment)
         self.setInst("v_cvt_pk_fp8_f32")
 
+class VCvtPkF32toBF8(VCvtInstruction):
+    def __init__(self, dst, src0, src1, vop3: Optional[VOP3PModifiers] = None, comment="") -> None:
+        super().__init__(CvtType.CVT_PK_F32_to_BF8, dst, [src0, src1], vop3, comment)
+        self.setInst("v_cvt_pk_bf8_f32")
+
 # V Mask
```

Comment:
Same as above comment, #564 . Probably this is not really mandatory but I'd suggest you do a rebase on latest develop.